Gemfile.lock

```diff
@@ -1069,7 +1069,7 @@ DEPENDENCIES
   web-console
 
 RUBY VERSION
-   ruby 3.0.6p216
+   ruby 3.0.7p220
 
 BUNDLED WITH
-   2.5.6
+   2.3.20
```

Comment:
This use the same bundler version than [decidim 0.27.6 uses](https://github.com/decidim/decidim/blob/v0.27.6/Gemfile.lock#L892).